app/js/sumo_db.js

```diff
@@ -163,7 +163,6 @@
       var endpoint = API_V2_BASE + 'question/';
       endpoint += '?product=' + PRODUCT;
       endpoint += '&is_solved=0';
-      endpoint += '&is_taken=0';
```

Comment:
Let's keep this in. We just want to deactivate the user facing bugs but we should not display taken questions in the "new" field.